guides/common/modules/proc_configuring-simplified-alternate-content-sources.adoc

```diff
@@ -3,6 +3,7 @@
 [id="Configuring_Simplified_Alternate_Content_Sources_{context}"]
 = Configuring simplified alternate content sources
 
+[role="_abstract"]
```

Comment:
Lacks an abstract.

How about

You can configure simplified alternate content sources (ACS) to download packages on {SmartProxyServers} from the upstream source.
Metadata will always come from your {ProjectServer}.